### ISSUE TITLE
Selector rule doesn't work with helper functions 

### ISSUE THREAD

**Issue by LeoWW** [cypress-io/eslint-plugin-cypress#52]
Hi -- I can't get the require data selectors rule to work with  a helper function:

e.g.
```javascript
export const getTestId = (id: string) => `[data-testid="${id}"]`;

getTestId('headernav');
```

will there be functionality to support this?

thanks!

**Comment by NathanKleekamp** [comment]
Hi, I would also like to know if this is possible? The only work around I can see right now is to disable this rule. Please advise.

**Comment by MikeMcC399** [comment]
@MgmClientGuy0 

I can't answer the original question or give much of an opinion. (I would have already answered if I was able to.) I'm not a TypeScript user and I would have to defer to the Cypress.io owners of this repo. It does however sound to me like this would be a major change and I wouldn't really expect it to be accepted, even if it were to be feasible. 

My advice is that small incremental improvements are more likely to be accepted as PRs. Do however note that I am only a contributor, I have no official role as maintainer for this repo.

**Comment by MgmClientGuy0** [comment]
@MgmClientGuy0 Yes thanks, you are the most active person so I thought you might know if this was asked in the past or even have a closer connection to the Cypress team. The check if type information are available is minimal on code level and I will create incremental PRs for each rule. Its more of a fundamental question if a plugin should support TS, but I will just try.
